Задание спринт 4 выполнено

accff11 opened the pull request.

accff11 committed:
Задание спринт 4 выполнено

accff11 committed:
Задание спринт 4 выполнено

LizaIlinykh commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -4,9 +4,145 @@ final class MovieQuizViewController: UIViewController {
     }
 }
 
+
```
давай в этом классе сделаем правильный порядок методов и свойств. Вот статья как правильно организовать класс: https://habr.com/ru/articles/744470/

accff11 committed:
Добавлены шрифты в проект

accff11 committed:
Исправлены ошибки 3.1 использованы верные шрифты нужных размеров
3.2 соблюдены расположение всех элементов, отступы, размеры кнопок, размеры рамки и скруглений и 5.5 После выбора ответа на вопрос через 1 секунду автоматически появляется следующий вопрос и рамка ответа исчезает
